force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +477,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        
+        if (person1.age > person2.age) {
+            return person1;
+        } else if (person2.age > person1.age) {
+            return person2;
+        } else {
+            return person1;
+        }
```

Comment:
How could we shorten this up dramatically? Hint: consider using `>=` in the first if statement. If we did that, how would the rest of it change?

Reply:
OK got that. Like the previous constructor example, I tend to follow the HW literally per the comments, but agree on this approach for production code for sure. :-)